lib/actions/user.js

```diff
@@ -148,7 +148,9 @@ export function createOrUpdateUserMonitoredTrip (tripData, isNew) {
 
       // TODO: improve the UI feedback messages for this.
       if (result.status === 'success' && result.data) {
-        alert('Your preferences have been saved.')
+        if (!silentOnSuccess) {
+          alert('Your preferences have been saved.')
+        }
```

Comment:
Why is the alert even part of the design here? To me its earlier inclusion felt like the result of early development, but I don't think it should be formalized in the code. Let's just rip the alert out.

Reply:
Moved the discussion to https://github.com/opentripplanner/otp-react-redux/issues/245